app/controllers/admin/partners_controller.rb

```diff
@@ -65,6 +65,13 @@ def update
       end
     end
 
+    def generate_manual_payout
+      # Generate report for the selected partner
+      publisher_ids = [].push(params[:id])
+      EnqueuePublishersForPayoutJob.perform_later(final: true, manual: true, publisher_ids: publisher_ids)
```

Comment:
`EnqueuePublishersForPayoutJob.perform_later(final: true, manual: true, publisher_ids: [params[:id]])`

Reply:
This is better, thanks Al @yachtcaptain23